src/python/bot/tasks/task_creation.py

```diff
@@ -227,3 +237,57 @@ def create_tasks(testcase):
   # Just create the minimize task for now. Once minimization is complete, it
   # automatically created the rest of the needed tasks.
   create_minimize_task_if_needed(testcase)
+
+
+def _get_commits(commit_range, job_type):
+  """Get commits from range."""
+  start, end = revisions.get_start_and_end_revision(commit_range)
+  components = revisions.get_component_range_list(start, end, job_type)
+  if ':' not in components[0]['link_text']:
+    return components[0]['link_text'], components[0]['link_text']
+
+  old_commit, new_commit = components[0]['link_text'].split(':')
+  if old_commit == '0':
+    old_commit = ''
+
+  return old_commit, new_commit
+
+
+def request_bisection(testcase, bisect_type):
+  """Request precise bisection."""
+  pubsub_topic = local_config.ProjectConfig().get('bisect_service.pubsub_topic')
+  if not pubsub_topic:
+    return
+
+  if bisect_type == 'fixed':
+    old_commit, new_commit = _get_commits(testcase.fixed, testcase.job_type)
+  elif bisect_type == 'regressed':
+    old_commit, new_commit = _get_commits(testcase.regression,
+                                          testcase.job_type)
+  else:
+    raise ValueError('Invalid bisection type ' + bisect_type)
+
+  reproducer = blobs.read_key(testcase.minimized_keys or testcase.fuzzed_keys)
+  target = testcase.get_fuzz_target()
```

Comment:
We need to skip blackbox fuzzer testcases or testcase that are not fuzz target, add a bailout.

Reply:
done.